### ISSUE TITLE
Jekyll Footnotes v0.7.0 breaks with Jekyll v3.1.2

### ISSUE THREAD

**Issue by se4u** [orangejulius/jekyll-footnotes#5]
the following fix on line 70 of `footnotes.rb` fixes it.

```
- converter = site.getConverterImpl(::Jekyll::Converters::Markdown)
+ converter = if site.respond_to?(:find_converter_instance)
                  site.find_converter_instance(::Jekyll::Converters::Markdown)
                else
                  site.getConverterImpl(::Jekyll::Converters::Markdown)
                end
```

See https://github.com/jekyll/jekyll/issues/3662 for details about the fix.

**Comment by orangejulius** [comment]
This would make a great pull request ;) ;)

On Tue, Apr 12, 2016, at 01:13 AM, se4u wrote:

> the following fix on line 70 of footnotes.rb fixes it.
> - converter = site.getConverterImpl(::Jekyll::Converters::Markdown) +
>   converter = if site.respond_to?(:find_converter_instance)
>   site.find_converter_instance(::Jekyll::Converters::Markdown) else
>   site.getConverterImpl(::Jekyll::Converters::Markdown) end
> 
> See jekyll/jekyll#3662[1] for details about the fix.
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub[2]

Links:
1. https://github.com/jekyll/jekyll/issues/3662
2. https://github.com/orangejulius/jekyll-footnotes/issues/5

**Comment by orangejulius** [comment]
Fixed in #6

**Comment by orangejulius** [comment]
Version 1.0.0 was just pushed which fixes this. Enjoy!
